platform/linux-generic/odp_classification.c

```diff
@@ -878,6 +881,28 @@ static inline int verify_pmr_ipv6_next_hdr(const _odp_ipv6hdr_t *ipv6, pmr_term_
 	return 0;
 }
 
+static inline int verify_pmr_ipv4_dscp(const _odp_ipv4hdr_t *ipv4, pmr_term_value_t *term_value)
```

Comment:
These could be simplified similarly to (suggested) `verify_pmr_ipv*_next_hdr()` functions.